### PR TITLE
[advanced audit]fix comment about throttle burst

### PR DESCRIPTION
**Pull request by CaoShuFeng**
/assign @sttts

**Release note**:
```release-note
NONE
```

### CODE REVIEW

File: staging/src/k8s.io/apiserver/plugin/pkg/audit/buffered/buffered.go [hunk 1]
```diff
@@ -54,7 +54,7 @@ type BatchConfig struct {
 	ThrottleEnable bool
 	// ThrottleQPS defines the allowed rate of batches per second sent to the delegate backend.
 	ThrottleQPS float32
-	// ThrottleBurst defines the maximum rate of batches per second sent to the delegate backend in case
+	// ThrottleBurst defines the maximum of request sent to the delegate backend at the same moment in case
```
Comment (CaoShuFeng) [inline staging/src/k8s.io/apiserver/plugin/pkg/audit/buffered/buffered.go]: We can use up all tokens in one moment, rather than in one second.

Comment (sttts) [reply staging/src/k8s.io/apiserver/plugin/pkg/audit/buffered/buffered.go]: requests

Comment (crassirostris) [inline staging/src/k8s.io/apiserver/plugin/pkg/audit/buffered/buffered.go]: `...maximum number of requests...`

**Review by crassirostris — COMMENTED** [review]
Well, these requests can be sent in different moments. I.e. considering qps is 10 and burst in 15, 8 requests, then 11, then again 8, then 12. 

Tough thing to phrase, yes

File: staging/src/k8s.io/apiserver/plugin/pkg/audit/buffered/buffered.go (hunk 1 above)
Comment (CaoShuFeng) [reply staging/src/k8s.io/apiserver/plugin/pkg/audit/buffered/buffered.go]: Done

**Pull request merged**